src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj

```diff
@@ -175,6 +175,7 @@
     <Compile Include="Compiler\ReadyToRunTableManager.cs" />
     <Compile Include="IBC\IBCProfileData.cs" />
     <Compile Include="IBC\IBCProfileParser.cs" />
+    <Compile Include="Perf\ReadyToRunPerfEventSource.cs" />
```

Comment:
Nit: do we expect more files to end up in the `Perf` directory? I would just dump it under the `Compiler` directory.

Reply:
Done. If we end up adding a lot more perf stuff we can remake the directory then.